### DISCUSSION TITLE
自己可以访问使用，别人用不了

### DISCUSSION THREAD

**Discussion by 180335265** [babaohuang/GeminiProChat#116] [General]
![image](https://github.com/babaohuang/GeminiProChat/assets/151026922/bae73e11-6407-4ba9-9b24-1cacd49759a6)

**Comment by gitlayzer** [comment]
Its API needs to be changed once a day, but when I changed it today, it showed that I did not have permission. Maybe Gemini’s API is no longer allowed to apply for it, or it may be my personal problem.
